ecommerce/maximizing_revenue/pricing.rst

```diff
@@ -1,31 +1,32 @@
 ==============================================
-How to adapt the prices to my website visitors
+How To Adapt The Prices To My Website Visitors
 ==============================================
 
-This section sheds some light on pricing features of eCommerce app:
+This section sheds light on pricing features found in the eCommerce app:
 
 * force a price by geo-localization,
 
 * let the customer choose the currency.
 
-As a pre-requisite, check out how to managing produt pricing:
+As a pre-requisite, check out how to managing product pricing:
 :doc:`../../sales/products_prices/prices/pricing`).
 
-Geo-IP to automatically apply the right price
-================================================
+Geo-IP Automatically Applies The Right Price
+============================================
 
-Assign country groups to your pricelists. That way, your visitors not yet
-logged in will get their own currency when landing on your website.
+Assign country groups to your pricelists. That way, visitors who aren't
+logged in yet will get their own currency when landing on your website.
 
 Once logged in, they get the pricelist matching their country.
 
-Currency selector
+Currency Selector
 =================
 
-In case you sell in several currencies, you can let your customers choose
-their own currency. Check *Selectable* to add the pricelist to the website drop-down menu.
+In the case that you sell in several currencies, you can let your customers choose
+their own. Check the *Selectable* box to add the pricelist to the website drop-down menu, which
+can be found in *Pricelists* under the *Products* menu, located in the Website application.
 
-.. image:: ./media/website_pricelist_select.png
```

Comment:
Is this image still used somewhere?

Reply:
I don't believe so. I'll delete it right away.